src/improvements/template/OPCMUpgradeV200.sol

```diff
@@ -0,0 +1,146 @@
+                absolutePrestate: absolutePrestates[chains[i].chainId]
+            });
+        }
+
+        // Before '_build()' is invoked by the 'build()' function in 'MultisigTask.sol', we start recording all state changes using 'vm.startStateDiffRecording()'.
+        // The primary purpose of '_build()' is to trigger the necessary account accesses so they are recorded when we call 'vm.stopAndReturnStateDiff()'.
+        // These recorded accesses are then converted into actions that the safe eventually executes via MultiCall3DelegateCall (in this case).
+        // More specifically, we want to ensure that the correct target and callData are recorded for the 'Call3' struct in MultiCall3DelegateCall.
+        // In this case, the 'target' should be the OPCM address, and the 'callData' should be the ABI-encoded 'upgrade()' call.
+        //
+        // The code below is unintuitive because we expect it to revert. Due to limitations in Foundry's 'prank' cheatcode (see: https://github.com/foundry-rs/foundry/issues/9990),
+        // we cannot correctly use a delegatecall to this function without it reverting.
+        //
+        // As a workaround, we make the call anyway to ensure the desired OPCM account access is recorded. This is acceptable because we later simulate
+        // the actual 'OPCM.upgrade()' call. However, it is crucial that the 'OPCM.upgrade()' call does not revert during the simulation.
+        (bool success,) = opcm().call(abi.encodeCall(IOPContractsManager.upgrade, (opChainConfigs)));
+        require(!success, "OPCMUpgradeV200: Call unexpectedly succeeded; expected revert due to non-delegatecall.");
+    }
+
+    /// @notice validate the task for a given l2chain
+    function _validate(VmSafe.AccountAccess[] memory, Action[] memory) internal view override {
+        SuperchainAddressRegistry.ChainInfo[] memory chains = superchainAddrRegistry.getChains();
+
+        for (uint256 i = 0; i < chains.length; i++) {
+            uint256 chainId = chains[i].chainId;
+            bytes32 currentAbsolutePrestate = Claim.unwrap(absolutePrestates[chainId]);
+            address proxyAdmin = superchainAddrRegistry.getAddress("ProxyAdmin", chainId);
+            address sysCfg = superchainAddrRegistry.getAddress("SystemConfigProxy", chainId);
+
+            IStandardValidatorV200.InputV200 memory input = IStandardValidatorV200.InputV200({
+                proxyAdmin: proxyAdmin,
+                sysCfg: sysCfg,
+                absolutePrestate: currentAbsolutePrestate,
+                l2ChainID: chainId
+            });
+
+            string memory reasons = STANDARD_VALIDATOR_V200.validate({_input: input, _allowFailure: true});
+            string memory expectedErrors_11155420 =
+                "PDDG-50,PDDG-DWETH-40,PDDG-ANCHORP-40,PLDG-50,PLDG-DWETH-40,PLDG-ANCHORP-40";
+            require(
+                keccak256(bytes(reasons)) == keccak256(bytes(expectedErrors_11155420)),
```

Comment:
is this list of expected errors true for all chains (whether mainnet or sepolia, whether OP, base, etc), or should this string be a config file input?